packages/components/addon/components/hds/app-footer/index.hbs

```diff
@@ -0,0 +1,8 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+<div class="hds-app-footer" ...attributes>
+  {{yield}}
```

Comment:
I still have mixed feeling between having a generic `{{yield}}` vs yielding only footer-related components like we do in the SideNav https://github.com/hashicorp/design-system/blob/main/packages/components/addon/components/hds/side-nav/list/index.hbs

Reply:
Resolving as this has been addressed.